backend/file-system/src/main/java/com/htec/filesystem/repository/FileRepository.java

```diff
@@ -25,6 +26,20 @@
 
     @Query("SELECT f " +
             "FROM FileEntity f JOIN ShelfEntity s ON (f.shelfId = s.id)" +
-            "WHERE s.userId = ?1 AND f.parentFolderId = ?2 AND f.isDeleted = ?3")
-    List<FileEntity> findAllByUserIdAndParentFolderId(Long userId, Long folderId, Boolean isDeleted);
+            "WHERE s.userId = :userId AND f.parentFolderId = :folderId AND f.isDeleted = :deleted")
+    List<FileEntity> findAllByUserIdAndParentFolderId(@Param("userId") Long userId,
```

Comment:
maybe findAllByUserIdAndParentFolderIdAndIsDeleted is better name cause your WHERE clause contains isDeleted?

Reply:
changed